src/routes/queues.ts

```diff
@@ -45,22 +39,26 @@ const getStats = async ({ queue }: BullBoardQueue): Promise<ValidMetrics> => {
   return validMetrics
 }
 
-const formatJob = async (job: Job | JobMq) => ({
-  id: job.id,
-  timestamp: job.timestamp,
-  processedOn: job.processedOn,
-  finishedOn: job.finishedOn,
-  progress: job.progress,
-  attempts: job.attemptsMade,
-  delay: await job.isDelayed(),
-  // failedReason: job.failedReason, // TODO: Apparently this doesn't exist
-  stacktrace: job.stacktrace,
-  opts: job.opts,
-  data: job.data,
-  name: job.name,
-})
-
-const statuses = [
+const formatJob = (job: Job | JobMq): app.AppJob => {
+  const jobProps = job.toJSON()
+
+  return {
+    id: jobProps.id,
+    timestamp: jobProps.timestamp,
+    processedOn: jobProps.processedOn,
+    finishedOn: jobProps.finishedOn,
+    progress: jobProps.progress,
+    attempts: jobProps.attemptsMade,
+    delay: job.opts.delay,
```

Comment:
This is a change from previous behaviour!
`delay` would be a boolean before, but would be used as a number in the components, so I'm changing it to use the actual delay instead of doing `+1` or `+0` depending of whether this is true or false 😅